app/__mocks__/@twilio/voice-react-native-sdk.ts

```diff
@@ -72,3 +72,11 @@ export const Call = {
     QualityWarningsChanged: 'qualityWarningsChanged',
   },
 };
+
+export const AudioDevice = {
+  Type: {
+    Earpiece: 'earpiece',
+    Speaker: 'speaker',
+    Bluetooth: 'bluetooth',
```

Comment:
Is this always true? Will we ever have more than this?

Reply:
This is it according to the native layers.